packages/reporters/liquidation-reporter/excel-writer.ts

```diff
@@ -0,0 +1,116 @@
+// This script takes in info from the liquidation reporter and builds an excel spreadsheet containing condensed information.
+
+const xl = require("excel4node");
+
+export function createExcelSheetFromLiquidationDrawDownData(drawDownData: any) {
+  // Create a new instance of a Workbook class
+  const wb = new xl.Workbook();
+
+  const boldStyle = wb.createStyle({ font: { bold: true } });
+
+  Object.keys(drawDownData).forEach((collateralAddress: string) => {
+    const collateralData = drawDownData[collateralAddress];
+    const sheet = wb.addWorksheet(collateralData.collateralSymbol);
+
+    // Heading
+    sheet
+      .cell(1, 1)
+      .string(`Collateral: ${collateralData.collateralSymbol}`)
+      .style({ font: { size: 16, bold: true } });
+
+    // Collateral info
+    sheet
+      .cell(2, 1)
+      .string("Collateral Price(USD)")
+      .style(boldStyle);
+    sheet.cell(2, 2).number(Number(collateralData.collateralPriceInUsd));
+    sheet
+      .cell(3, 1)
+      .string("Collateral TVL(USD)")
+      .style(boldStyle);
+    sheet.cell(3, 2).number(Number(Number(collateralData.collateralValueInUsd).toFixed(2)));
+    sheet
+      .cell(4, 1)
+      .string("Collateral Address")
+      .style(boldStyle);
+    sheet.cell(4, 2).string(collateralAddress);
+
+    // Financial contracts list
+    sheet
+      .cell(2, 7)
```

Comment:
Why does this start at row(?) 7?

Reply:
the content is off on the right of the sheet.